modules/check_in/app/services/travel_claim/claim_submission_service.rb

```diff
@@ -383,19 +396,30 @@ def error_template_id
     def determine_error_template_id(error)
       if error.is_a?(Common::Exceptions::BackendServiceException) &&
          error.response_values[:detail]&.include?('already been created')
-        @facility_type&.downcase == 'oh' ? CheckIn::Constants::OH_DUPLICATE_TEMPLATE_ID : CheckIn::Constants::CIE_DUPLICATE_TEMPLATE_ID
+        template_for_duplicate_error
       else
         error_template_id
       end
     end
 
+    ##
+    # Returns duplicate template ID based on facility type
+    #
+    def template_for_duplicate_error
+      if @facility_type&.downcase == 'oh'
+        @context_constants::OH_DUPLICATE_TEMPLATE_ID
+      else
+        @context_constants::CIE_DUPLICATE_TEMPLATE_ID
+      end
+    end
+
     ##
     # Increments the appropriate success metric based on facility type
     #
     def increment_success_metric
       increment_metric_by_facility_type(
-        CheckIn::Constants::CIE_STATSD_BTSSS_SUCCESS,
-        CheckIn::Constants::OH_STATSD_BTSSS_SUCCESS
+        @context_constants::CIE_STATSD_BTSSS_SUCCESS,
+        @context_constants::OH_STATSD_BTSSS_SUCCESS
       )
```

Comment:
If the intention is for this service to be useable from a different context, the new standalone travel-pay app, I think we wouldn't want to increment check-in metrics outside of the check-in context. To make this dynamic we might have to restructure the `CheckIn::Constants` in a way that is consistent with the new app's constants.